parsec/core/backend_connection/cmds.py

```diff
@@ -121,6 +122,12 @@ async def _send_cmd(transport: Transport, serializer, **req) -> dict:
 
 ###  Backend authenticated cmds  ###
 
+### Organization API ###
+
+
+async def organization_stats(transport: Transport) -> dict:
```

Comment:
Naming should be `authenticated_organization_stats` and `apiv1_administration_organization_stats`

In theory we should have a better naming convention to avoid mixing command between authentications type and api versions, but I think it's a big task that would be better outside of this PR

Reply:
done

Reply:
Undone it, it was creating problems with the `expose_cmds.py` functions. As you said, it should be an entire PR to improve the naming